### DISCUSSION TITLE
[PC/PSP] Call of Duty weapons sounds

### DISCUSSION THREAD

**Discussion by capuccin0** [nzp-team/nzportable#863] [Other Content Releases]
### Replaces the vanilla NZP weapon sounds with sounds from the original Call of Duty and its expansion pack United Offensive

_Weapons which sounds have not been replaced:_
- Ray Gun (it was never planned to be in CoD until Treyarch came up with Nazi Zombies)
- DG-2 (same reason)
- Double-Barrel Shotgun/Trench Gun (there's no shotguns in CoD and CoD: United Offensive)
- Flamethrower (it is present in CoD: United Offensive, but I couldn't figure out how to make it sound ok in NZP)
- Knife (same reason as with shotguns)
- MP5K/Ballistic Knife (CoD is a WW2 game, there's no modern weapons in there, obviously)

_Some workarounds I used:_
- MG42 uses reload sounds from MG34, since MG42 is a mounted weapon in CoD
- PTRS uses reload sounds from BAR, same reason.
- Type 100 uses Sten sounds
- 357 uses Webley sounds
- Since Springfield and Kar98k share the same sounds in NZP, I decided to use Kar98k sounds because Springfield sucks.
- Ping sound of the M1 Garand is not taken from CoD, but from the original stock sound, since in CoD the sound of the ping is mixed with the fire sound and used as the sound of the last shot.

[**Download**](https://github.com/user-attachments/files/16554057/cod_weapon_sfx_v2.zip)

Let me know, if this small sound pack works with other NZP ports

**Comment by owenjames2023** [comment]
Thanks this works well for my psp!

**Comment by spectrum1-wave** [comment]
does this work with nzp.gay browser?

**Reply by capuccin0** [reply]
No. As far as I know, you can't load mods in browser version
